app/views/idv/_acuant_sdk_selfie_capture_form.html.erb

```diff
@@ -0,0 +1,58 @@
+<% if Figaro.env.acuant_sdk_document_capture_enabled == 'true' %>
+  <% content_for :meta_tags do %>
+    <meta
+      name='acuant-sdk-initialization-endpoint'
+      content='<%= Figaro.env.acuant_sdk_initialization_endpoint %>'
```

Comment:
what about using rails helpers to build these tags so we don;'t ahve to worry about escaping?

```ruby
tag :meta, name: 'acuant-sdk-initialization-endpoint', content: Figaro.env.acuant_sdk_initialization_endpoint
```

Reply:
yes good stuff!